dcp-0006/dcp-0006.mediawiki

```diff
@@ -0,0 +1,345 @@
+# Users need to be able to send utxos to the treasury account. This is a coin burn that is analogous to purchasing a ticket. This feature is also going to be used to move currently unused funds in the treasury to the new treasury account.
+# Spending treasury funds now requires stakeholders to vote on the expenditure. The Pi key operators creates a "treasury spend" transaction and signs it. This transaction is then broadcast onto the network and it enters the mempool. Voting wallets will detect this transaction and they will automatically vote Yes/No on the expenditure based on the embedded key and signature (if enabled). Treasury expenditure votes are collected during the consensus voting process. If the treasury spend transaction collects enough yes votes it is included in the chain and payees will receive their funds.
+
+==Specification==
+
+The following section will detail the new transaction types.  All new
+transactions shall be part of the stake tree and all treasury opcodes shall be
+rejected if used in scripts in the regular tree.
+
+===Treasurybase===
+
+The newly introduced treasurybase moves the developer subsidy from the
+coinbase transaction to its own transaction in the stake tree. The
+treasurybase transaction shall be the first transaction in the stake tree.
+There shall be no more than one treasurybase transaction per block. The
+developer subsidy shall credit the treasury account in full unlike the current
+implementation which credits a proportion based one the number of tickets
+included in the block.
+
+A treasurybase transaction is identified by the following characteristics:
+* The wire message transaction version shall be 3.
+* One TxIn that shall *NOT* have a signature script.
+* All TxOut scripts shall be version 0.
+* There must be two TxOut scripts.
+** TxOut[0] script must be OP_TADD
+** TxOut[1] shall be an OP_RETURN followed by a twelve byte datapush OP_DATA_12 and corresponding data for its script.
+* The previous transaction output point shall be null.
+
+{|
+!Transaction!!Script!!Description
+|-
+|TxIn[0]||No script||Stakebase and ValueIn shall be exact development subsidy
+|-
+|TxOut[0]||[0]OP_TADD||Value shall be exact development subsidy
+|-
+|TxOut[1]||[0]OP_RETURN<br>[1]OP_DATA_12<br>[2:6][4]{LE encoded height}<br>[6:14][8]{random}||Block height shall be little endian encoded in [2:6] script bytes
+|}
+
+Additional consensus enforced characteristics:
+
+Note that some rules have been enforced prior the ones listed but out of an
+abundance of caution some tests are repeated in places where it makes sense.
+This is normal and desired.
+
+* First transaction of the stake tree shall be a treasurybase.
+* There shall be only one treasurybase transaction in the stake tree.
+* The previous out point for TxIn[0] shall be null.
+* The treasurybase transaction fraud proof shall be null.
+* The length of the signature script of TxIn[0] shall be zero.
+* The treasurybase transaction shall have more than 1 outputs.
```

Comment:
make this more clear ? Number and plural

Reply:
This looks right to me. Anyone has a suggestion on how to phrase this differently?